nanome_rmsd/RMSD.py

```diff
@@ -88,18 +88,26 @@ def on_complexes_received(self,complexes):
         target_complex = complexes[0]
         mobile_complex = complexes[1:]
         result = 0
+
+        total_percentage = len(self._mobile)
+        percentage_count = 0
+
         for x in mobile_complex:
             result += self.align(target_complex, x)
+            percentage_count += 1/total_percentage
```

Comment:
it's generally better to increment and int and then divide by total when setting.

percentage_count +=1
self._menu.change_loading_percentage(percentage_count/percentage_total)

This is because floats don't add up exactly the way you expect. 1/3 + 1/3 + 1/3 != 1.
In this case it doesn't matter because the error would be visibly indistinguishable, but just something to keep in mind.
You don't need to bother changing this.